Update registry k8s.gcr.io -> registry.k8s.io (#172)
kubernetes/committee-security-response: severity-ratings.md

- Components built from source in the `kubernetes` github organization and
  [all current organizations in use](https://git.k8s.io/community/github-management/README.md#actively-used-github-organizations)

> - Container images under the `registry.k8s.io` repository

Reviewer: We will support both registries in regards to vulnerabilities.  Removing the reference to the old registry is not valid.

> - Container images under the `k8s.gcr.io`(to be deprecated) -> `registry.k8s.io` repository

Reviewer: Both are supported indefinitely:

```suggestion
- Container images under the `k8s.gcr.io` and `registry.k8s.io` repositories
```
